src/client/src/App.js

```diff
@@ -15,16 +16,21 @@ const theme = createTheme({
       main: "#7ca0e5",
```

Comment:
Is it fine to create color in js, not css? By the way, I like to have main colors in variables:
```
:root {
  --dark-green: #21825B;
  --light-green: #36D695;
}
```
And then use them like `color: var(--dark-green);`